src/sqlancer/postgres/PostgresGlobalState.java

```diff
@@ -76,6 +78,18 @@ public void setConnection(SQLConnection con) {
         return opClasses;
     }
 
+    private List<String> getTableAccessMethods(SQLConnection con) throws SQLException {
+        List<String> tableAccessMethods = new ArrayList<>();
+        try (Statement s = con.createStatement()) {
+            try (ResultSet rs = s.executeQuery("SELECT amname FROM pg_am WHERE amtype = 't';")) {
```

Comment:
a small comment explaining that the `amtype` table contains both the index access methods as well as the table access methods and that the table access methods are marked with `amtype = 't'` would help a non-fluent postgres reader.

Due to postgres using a single character to encode this makes the query not too easy to understand for someone that is not intimately familiar with postgres internals.